modules/virt-defining-storageclass.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-preparing-cdi-scratch-space.adoc
+
+[id="virt-defining-storageclass_{context}"]
+= Defining a storage class
+
+To specify the storage class that the Containerized Data Importer (CDI) uses when allocating scratch space, add the `spec.scratchSpaceStorageClass` field to the `HyperConverged` custom resource (CR).
```

Comment:
I prefer the same verb in the header and introduction. So I might phrase it like this:

You can define the storage class that the CDI uses when allocating scratch space by adding the spec.etc parameter to the HyperConverged CR.

or 

To define the storage class ..., add the spec.scratchSC parameter to the CR.